lib/server.js

```diff
@@ -42,8 +42,10 @@ module.exports = (config, callback) => {
     // Create a server with a host and port
     const server = new Hapi.Server({
         app: {
-            datastore: config.datastore,
-            executor: config.executor
+            pipelineFactory: config.pipelineFactory,
```

Comment:
I'll update the docs.

`lib/server` is already dependent on `bin/server` to pass datastore and executor. Since datastore and executor are needed only by the factory methods, it makes sense to pass the factories in the same way.